lib/rspec/core/formatters/json_formatter.rb

```diff
@@ -60,8 +60,7 @@ def dump_profile(profile)
         # @api private
         def dump_profile_slowest_examples(profile)
           @output_hash[:profile] = {}
-          sorted_examples = profile.slowest_examples
-          @output_hash[:profile][:examples] = sorted_examples.map do |example|
+          @output_hash[:profile][:examples] = profile.slowest_examples.map do |example|
```

Comment:
It's fine.  I was just making sure there wasn't some behavioral change here I wasn't noticing.